test/server/services/user.js

```diff
@@ -395,5 +395,35 @@ describe('User service tests', function() {
           done();
         }, done);
     });
+    it('should return a null call_sid for userManageParams2', (done) => {
+      usersService.getUserById({ id: this.userManageParams2.id })
+        .then((user) => {
+          expect(user.attributes.call_sid).to.be.null;
+          done();
+        }, done);
+    });
+    it('should update the call_sid for userManageParams2', (done) => {
+      const call_sid = 'CA1234567890qwertyuiopasdfghjklzxc';
+      const params = {
+        call_sid,
+        id: this.userManageParams2.id
+      };
+      usersService.updateUserById(params)
+        .then((user) => {
+          expect(user.attributes.call_sid).to.equal(call_sid);
```

Comment:
Wondering if we need to assert that other fields were not changed. Did you at least do this manually?

Reply:
I checked manually using Postman and the db. I'll update the tests.